src/main/resources/api.yml

```diff
@@ -0,0 +1,503 @@
+          examples:    # Child of media type
+            Ada:   # Example 1
+              value:
+                firstName: Ada
+                lastName: Polkanova
+                eMail: dogs4fun@dogmail.com
+                phone: +49 177 77777777
+                serviceLocation: Berlin
+                description: Hi, I'm Ada, the dog.
+                photo: image
+    responses:
+      "200":
+        description: Result of the operation
+        schema:
+          type: long
+
+      default:
+        description: Unexpected error
+        schema:
+          $ref: '#/definitions/Error'
+  put:
+    summary: Updates a business profile
+    description: The endpoint updates details of business profile previously retrieved from the database, containing content of the following fields: id, first name, last name, e-mail, phone, service location, short service description and an image.
+                   The response includes the result of the operation.
+    requestBody:
+      content:
+        application/json:     # Media type
+          schema:             # Request body contents
+            $ref: '#/components/schemas/Business'   # Reference to an object
+          examples:    # Child of media type
+            Ada:   # Example 1
+              value:
+                id: 10
+                firstName: Ada
+                lastName: Polkanova
+                eMail: dogs4fun@dogmail.com
+                phone: +49 177 77777777
+                serviceLocation: Berlin
+                description: Hi, I'm Ada, the dog.
+                photo: image
+    responses:
+      "200":
+        description: Result of the operation
+        schema:
+          type: long
+
+      "403":
+        description: Unauthorized
+        schema:
+          $ref: '#/definitions/Error'
+
+      default:
+        description: Unexpected error
+        schema:
+          $ref: '#/definitions/Error'
+
+  /business/{id}:
+    get:
+      summary: Fetch a business profile
+      description: The endpoint returns information about a business profile with a given id.
+      The response includes the name and other details about the business.
+      parameters:
+      - name: id
+        in: url
+        description: Profile id
+        required: true
+        type: string
+      security:
+      - token: []
+      tags:
+      - business profile
+      responses:
+        "200":
+          description:  Profile fields
+          schema:
+            type: json
+            profile:
+              $ref: '#/definitions/Business'
+        "403":
+          description: Unauthorized for the current operation
+          schema:
+            $ref: '#/definitions/Error'
+        "404":
+          description: Profile not found
+          schema:
+            $ref: '#/definitions/Error'
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+    /orders:
+      get:
```

Comment:
Question: can we have an order outside of the context of the business?
If not, then what if the url looks like this:
`/business/{id}/orders`

Reply:
If a client wants to open his/her order to check its status (even before the order is taken by any of the businesses), it would be a different context, right?